eZ/Bundle/EzPublishCoreBundle/Command/ResizeOriginalImagesCommand.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Search\SearchHit;
+use eZ\Publish\Core\FieldType\Image\Value;
+use eZ\Publish\Core\IO\IOServiceInterface;
+use Liip\ImagineBundle\Binary\BinaryInterface;
+use Liip\ImagineBundle\Exception\Imagine\Filter\NonExistingFilterException;
+use Liip\ImagineBundle\Model\Binary;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ResizeOriginalImagesCommand extends ContainerAwareCommand
+{
+    const DEFAULT_ITERATION_COUNT = 25;
+
+    protected function configure()
+    {
+        $this->setName('ezplatform:resize-original-images')->setDefinition(
+            [
+                new InputArgument('contentTypeIdentifier', InputArgument::REQUIRED, 'Indentifier of ContentType which has ezimage FieldType.'),
+                new InputArgument('imageFieldIdentifier', InputArgument::REQUIRED, 'Identifier of field of ezimage type.'),
+                new InputArgument('variation', InputArgument::OPTIONAL, 'Variation which will be used for original images.', 'original'),
+                new InputArgument('iteration-count', InputArgument::OPTIONAL, 'Iteration count. Number of images to be recreated in a single iteration, for avoiding using too much memory.',
+                    self::DEFAULT_ITERATION_COUNT),
+            ]
+        );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $contentTypeIdentifier = $input->getArgument('contentTypeIdentifier');
+        $imageField = $input->getArgument('imageFieldIdentifier');
+        $variation = $input->getArgument('variation');
+        $iterationCount = (int)$input->getArgument('iteration-count');
+
+        $repository = $this->getContainer()->get('ezpublish.api.repository');
+        $contentTypeService = $repository->getContentTypeService();
+        $searchService = $repository->getSearchService();
+        $filterManager = $this->getContainer()->get('liip_imagine.filter.manager');
+        $ioService = $this->getContainer()->get('ezpublish.fieldType.ezimage.io_service.published');
+        $extensionGuesser = $this->getContainer()->get('liip_imagine.extension_guesser');
+
+        $contentType = $contentTypeService->loadContentTypeByIdentifier($contentTypeIdentifier);
+        $query = new Query();
+        $query->filter = new Query\Criterion\ContentTypeIdentifier($contentType->identifier);
+
+        $totalCount = $repository->sudo(function () use ($query, $searchService) {
```

Comment:
Please add an extra `user` option and set proper User Reference instead in the overridden `initialize` method (example could be found in some other command... ping me if needed).